### PR TITLE
add dashboard scaffold

### PR DESCRIPTION
**Pull request by aseaday**
Signed-off-by: nullday <aseaday@hotmail.com>

### CODE REVIEW

**Comment by VoVAllen** [comment]
I have a frontend PoC at 
demo: https://envd-dashboard-ui-demo.netlify.app/
Code: https://github.com/VoVAllen/envd-dashboard-ui

**Comment by aseaday** [comment]
LGTM, I'll try to add your code and test it

**Review by VoVAllen — COMMENTED** [review]
Is it possible to build the backend separately now? Also I think usually dist folder should not be included in git repo?

**Comment by aseaday** [comment]
> 

Let me have a try

**Review by gaocegege — COMMENTED** [review]
I do not have the ability to review this PR

I will call @VoVAllen and @kemingy 

Thanks for your contribution! :tada: :+1:

File: dashboard/env.d.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+/// <reference types="vite/client" />
```
Comment (kemingy) [inline dashboard/env.d.ts:1]: What's the purpose of this file?

**Review by VoVAllen — DISMISSED** [review]
LGTM. Did you solve the goreleaser problem?

**Comment by VoVAllen** [comment]
There's a pre hook in goreleaser https://goreleaser.com/customization/build/

**Comment by aseaday** [comment]
> There's a pre hook in goreleaser https://goreleaser.com/customization/build/

I think I could add steps before goreleaser in github workflow.

**Comment by aseaday** [comment]
We can step forward to merge this first. I will open another pr to solve release problem. @gaocegege @VoVAllen

**Comment by VoVAllen** [comment]
@aseaday Can you fix the lint error? Other part looks good to me. I think we can merge it once lint is fixed

**Comment by VoVAllen** [comment]
Thanks!

**Pull request merged**